### ISSUE TITLE
constexpr for explicit constructors of vectors and matrices

### ISSUE THREAD

**Issue by RnMss** [g-truc/glm#493]
e.g.
 
```c++
namespace glm
{
    template <typename T, precision P>
    struct tvec2 {
        ...
        //from 
        //GLM_FUNC_DECL tvec2(T const & s1, T const & s2);`
        //to 
        GLM_FUNC_DECL GLM_CONSTEXPR tvec2(T const & s1, T const & s2);
        ...
    };
}
```

**Comment by Trass3r** [comment]
...and quaternions :)

**Comment by Groovounet** [comment]
This issue should be fixed in master branch for GLM 0.9.8.

Thanks for reporting,
Christophe
